pyteal/__init__.pyi

```diff
@@ -76,6 +76,7 @@ __all__ = [
     "EcdsaRecover",
     "EcdsaVerify",
     "Ed25519Verify",
+    "Ed25519Verify_Bare",
```

Comment:
@jdtzmn Minor:  After approving I realized I missed user guide consideration.  Can you extend https://pyteal.readthedocs.io/en/stable/crypto.html to include `Ed25519Verify_Bare`?